regolith/tools.py

```diff
@@ -375,6 +375,60 @@ def filter_projects(projects, authors, reverse=False):
     projs.sort(key=id_key, reverse=reverse)
     return projs
 
+def filter_grants_copi(input_grants, names, pi=True, reverse=True, multi_pi=True):
```

Comment:
fyi, this is probably the wrong approach.  If you are cutting/pasting code it usually means it is better to reuse or adapt the function that you copy/pasted from.  It is much easier to maintain a code-base where there is less code, and as little as possible repeated code....